org.eclipse.xtext.dev-bom/build.gradle

```diff
@@ -0,0 +1,105 @@
+        api "org.eclipse.jdt:org.eclipse.jdt.core:3.20.0"
+        /*
+        When adjusting the `lsp4j` version here, do not forget to do the following: 
+        - Update the version in `IdeProjectDescriptor#pom`.
+        - Update the test expectations by running `CliWizardIntegrationTest#main`.
+        - Update the versions in the 
+          `/org.eclipse.xtext.ide/META-INF/MANIFEST.MF` and 
+          `/org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF` files.
+        - Update Oomph releng/org.eclipse.xtext.contributor/Xtext.setup
+        */
+        api "org.eclipse.lsp4j:org.eclipse.lsp4j.generator:0.8.1"
+        api "org.eclipse.lsp4j:org.eclipse.lsp4j.jsonrpc:0.8.1"
+        api "org.eclipse.lsp4j:org.eclipse.lsp4j.websocket:0.8.1"
+        api "org.eclipse.lsp4j:org.eclipse.lsp4j:0.8.1"
+        api "org.eclipse.platform:org.eclipse.core.commands:3.9.600"
+        api "org.eclipse.platform:org.eclipse.core.contenttype:3.7.500"
+        api "org.eclipse.platform:org.eclipse.core.expressions:3.6.600"
+        api "org.eclipse.platform:org.eclipse.core.filesystem:1.7.600"
+        api "org.eclipse.platform:org.eclipse.core.jobs:3.10.600"
+        api "org.eclipse.platform:org.eclipse.core.resources:3.13.600"
+        api "org.eclipse.platform:org.eclipse.core.runtime:3.17.0"
+        api "org.eclipse.platform:org.eclipse.equinox.app:1.4.300"
+        api "org.eclipse.platform:org.eclipse.equinox.common:3.10.600"
+        api "org.eclipse.platform:org.eclipse.equinox.preferences:3.7.600"
+        api "org.eclipse.platform:org.eclipse.equinox.registry:3.8.600"
+        api "org.eclipse.platform:org.eclipse.osgi:3.15.100"
+        api "org.eclipse.platform:org.eclipse.text:3.10.0"
+        api "org.eclipse.xpand:org.eclipse.xpand:2.0.0"
+        api "org.eclipse.xpand:org.eclipse.xtend.typesystem.emf:2.0.0"
+        api "org.eclipse.xpand:org.eclipse.xtend:2.0.0"
+        api "org.eclipse.xtext:xtext-antlr-generator:2.1.1"
+        api "org.junit.jupiter:junit-jupiter-api:5.1.0"
+        api "org.junit.jupiter:junit-jupiter-engine:5.1.0"
+        api "org.ow2.asm:asm-commons:7.2"
+        api "org.ow2.asm:asm-tree:7.2"
+        api "org.ow2.asm:asm:7.2"
+    }
+}
+
+publishing {
+    publications {
+        LocalMavenRepo(MavenPublication) {
+            from components.javaPlatform
+            artifactId "xtext-dev-bom"
+            pom {
+                name = 'Eclipse Xtext Developer BOM'
+                description = 'Xtext Third Party Dependencies (Bill of Materials)'
+                url = 'https://www.eclipse.org/Xtext/'
+                licenses {
+                    license {
+                        name = 'Eclipse Public License, Version 1.0'
+                        url = 'http://www.eclipse.org/legal/epl-v10.html'
+                    }
+                }
+                scm {
+                    connection = "scm:git:git@github.com:eclipse/${rootProject.name}.git"
```

Comment:
git URL does not work for anonymous user (permission denied), use HTTPS URL:
`connection = "scm:git:https://github.com/eclipse/${rootProject.name}.git`

Reply:
please open a different issue. this affects all gradle stuff

Reply:
https://github.com/eclipse/xtext/issues/1652